packages/shared-components/cypress/e2e/footer/footer.cy.js

```diff
@@ -37,4 +37,66 @@ describe('Footer', () => {
       .should('have.text', 'Ontotext AD')
       .should('have.attr', 'href', 'http://ontotext.com');
   });
+
+  it('Should show/hide cookie consent component', () => {
+    // Given I visit the footer page
+    FooterSteps.visit();
+
+    // Then, I expect the cookie consent component not to be visible,
+    // because tracking is not allowed. Also, there is no license information
+    FooterSteps.getCookieConsentComponent().should('not.exist');
+
+    // When, I set a free license, prod mode and a user, who hasn't accepted the cookie policy
+    FooterSteps.setFreeLicense();
+    FooterSteps.setProdMode();
+    FooterSteps.setNotAcceptedUser();
+
+    // Then, I expect the cookie consent component to be visible, because we meet
+    // all 3 criteria for visibility
```

Comment:
I like how you describe steps the test is following. This makes them easier to follow